sensing/autoware_pointcloud_preprocessor/schema/approximate_downsample_filter_node.schema.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for Approximate Downsample Filter Node",
+  "type": "object",
+  "definitions": {
+    "approximate_downsample_filter": {
+      "type": "object",
+      "properties": {
+        "voxel_size_x": {
+          "type": "number",
+          "description": "voxel size along the x-axis [m]",
+          "default": "0.3",
+          "minimum": 0
```

Comment:
As I explained in previous PRs, please keep consistency with the floating representation

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/8480/commits/38a1aaacaffaeaeb6c37e5310d22a3d5d6885b93